node/typings.d.ts

```diff
@@ -119,11 +119,20 @@ interface Organization {
   id: string
   name: string
   tradeName?: string
+  collections: Collection[]
```

Comment:
same thing about this file (belongs to this other [PR](https://github.com/vtex-apps/b2b-organizations-graphql/pull/127))